SWIPs/swip-adaptive-node-protocol.md

```diff
@@ -0,0 +1,96 @@
+---
+SWIP: <to be assigned>
+title: Adaptive nodes capabilities protocol
+author: Louis Holbrook @nolash <rhabdo@ethswarm.org>, Tim Bansemer (editorial) @FantasticoFox <tim@ethswarm.org>
+status: Draft
+type: Track Networking
+created: 2019-07-11
+---
+
+## Simple Summary
```

Comment:
I would suggest that we merge `Simple Summary` and `Abstract` into one heading. As long as a sufficiently descriptive title is chosen, I don't see a need for both.